lms/templates/instructor/instructor_dashboard_2/certificate-invalidation.underscore

```diff
@@ -3,9 +3,16 @@
 </p>
 
 <div class="add-certificate-invalidation">
-    <input class='student-username-or-email' id="certificate-invalidation-user" type="text" placeholder="<%- gettext('Username or email address') %>">
-    <textarea class='notes-field' id="certificate-invalidation-notes" rows="10" placeholder="<%- gettext('Add notes about this learner') %>"></textarea>
+    <div>
+        <label for="certificate-invalidation-user"><%- gettext('Username or email address') %></label>
```

Comment:
I'd like to keep the scope of work limited to "adding missing labels and addressing any regression as a result of those additions".

I'm happy to create a follow-up story that dives in and does a thorough cleanup of these files that can go alongside the tabbed work. Would that work?